client/components/domains/domain-search-results/index.jsx

```diff
@@ -313,6 +316,13 @@ class DomainSearchResults extends React.Component {
 					/>
 				);
 			}
+
+			domainSkipSuggestion = (
+				<DomainSkipSuggestion
+					selectedSiteSlug={ this.props.selectedSite?.slug }
```

Comment:
The optionality here is pretty hacky (read brittle), I added it to prevent an error in the signup flow before a site exists this works because that flow also issues a suggestion request `include_wordpressdotcom=true` and would have a `showSkipButton = false` prop.

Seeing this again, its probably better to check the showSkipButton before even creating the element but it shouldn't really cause any problems :crossed_fingers: